src/Middleware/Diagnostics.EntityFrameworkCore/src/DatabaseExceptionFilter.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.Views;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore
+{
+    public class DatabaseExceptionFilter : IDeveloperPageExceptionFilter
+    {
+        private readonly ILogger _logger;
+        private readonly DatabaseErrorPageOptions _options;
+
+        public DatabaseExceptionFilter(ILogger<DatabaseExceptionFilter> logger, IOptions<DatabaseErrorPageOptions> options)
```

Comment:
Can you turn on nullability on this type?

Reply:
Hmm odd this isn't done automatically already.